### DISCUSSION TITLE
Duplicate IP addresses; can't see newly created(?) public servers

### DISCUSSION THREAD

**Discussion by phaves** [jamulussoftware/jamulus#1906] [Technical Support & Help]
For testing purposes, I created two Genre Rock public servers, one on my Mac (MojoM4--Private) and one using Windows (MM_PC_test). They both have the same IP address(71.204.133.190) but different ports. When I Connect on the Mac I see both servers in the list. On Windows, I only see the server I created under Windows. When I try to connect by specifying the IP address, I get TRYING TO CONNECT but no connection is made,  and I get the same behavior on the Mac.

I see that there is another pair of Genre Rock public servers with the same IP address (TestJam and PrivateKabaal, both Linux) and a pair of Any Genre 1 servers (Dazhi 1 and Dazhi 4, both Windows). There may be more - I haven't done an exhaustive check.

A, possibly related, problem is that when my band-mate attempted to set up a public server (see attached image), neither he nor I can see it in the database at [https://explorer.jamulus.io/](url), nor can we see it in the Connect list on Jamulus - any ideas? (We are trying to set up different public servers from our own computers to try and understand any variations in performance when public servers are hosted by different computers.)

![DJ_JamServer_setup](https://user-images.githubusercontent.com/9658275/125217992-cc81a980-e276-11eb-97c6-d1c1f100137a.jpeg)

Thanks in advance for your help,

Philip Haves

**Comment by dcorson-ticino-com** [comment]
I can see both of your servers and connect to them, no problems, except 165ms ping :)
Your problem must be some configuration of your router.
